Jenkinsfile

```diff
@@ -178,6 +179,15 @@ node('ubuntu-zion') {
     OsTools.runSafe(this, "docker system prune -a -f")
     OsTools.runSafe(this, 'git clean -f && git reset --hard origin/master')
   }
+
+  stage('Trigger Red Hat Certified Image Build') {
+    withCredentials([
+        string(credentialsId: 'docker-nexus3-rh-build-project-id', variable: 'PROJECT_ID'),
+        string(credentialsId: 'rh-build-service-api-key', variable: 'API_KEY')]) {
+      def redHatVersion = "${version}-ubi"
+      runGroovy('TriggerRedHatBuild.groovy', [redHatVersion, PROJECT_ID, API_KEY].join(' '))
```

Comment:
if you like you can use a relative path here

Reply:
Are you suggesting `./TriggerRedHatBuild.groovy` is better?

Reply:
no .. just leave it where it is or maybe `ci/TriggerRedHatBuild.groovy' or something like that.

Reply:
ah, excellent: just the advice I was hoping to get!
👍

Reply:
done!